modules/exploits/windows/misc/crosschex_device_bof.rb

```diff
@@ -0,0 +1,81 @@
+
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+  PAYLOAD_MAX_LENGTH = 8947
```

Comment:
If you use the `Space` key in the payload info hash, only payload modules that will fit will be available to the user and you won't have to check later on.

Reply:
Done, I always wondered what that option meant 😅